### DISCUSSION TITLE
help changing profile picture

### DISCUSSION THREAD

**Discussion by SkunkaMunka** [Atlas-OS/Atlas#1275] [Help & Support]
how do I change the atlas user picture? 
whenever I try to click on my account in the settings app nothing happens and clicking on "your account info" just takes me to the settings main menu, and I don't see the option to change the pic in control panel either

thanks in advance
![userpic](https://github.com/user-attachments/assets/459a0bcc-c2ec-4a13-8f7c-6d8e6fd5d475)

**Comment by fardiansyah1808** [comment]
I find it in Settings > Accounts > Your info.

But got an error while change the picture, error code: 0x88982f8b

**Reply by meetox80** [reply]
Can you please try manually replacing user.bmp in `C:\ProgramData\Microsoft\User Account Pictures`

[PFP.ps1](https://github.com/Atlas-OS/Atlas/blob/main/src/playbook/Executables/PFP.ps1)
